src/Action/AddAction.php

```diff
@@ -166,4 +169,18 @@ protected function _error(Subject $subject)
         $this->setFlash('error', $subject);
         $this->_trigger('beforeRender', $subject);
     }
+
+    /**
+     * Stopped callback
+     *
+     * @param \Crud\Event\Subject $subject Event subject
+     * @return \Cake\Network\Response
+     */
+    protected function _stopped(Subject $subject)
+    {
+        $subject->set(['success' => false]);
```

Comment:
wonder if success should default false, but $subject->success could be used to modify it - stopping it could also mean success in some cases?